src/server.js

```diff
@@ -23,8 +25,55 @@ const sendUserError = (err, res) => {
   }
 };
 
+const validateNameAndPass = (req, res, next) => {
+  const username = req.body.username;
+  const password = req.body.password;
+  if (!username || !password) {
+    res.status(STATUS_USER_ERROR);
+    res.json({ error: 'Must provide username and password' });
+    return;
```

Comment:
Calling `res.json` and other `res` functions like `res.send` performs an implicit `return`, so calling `return` afterwards is extraneous.